test/functional/api/cas/cli.py

```diff
@@ -109,6 +109,8 @@ def print_statistics_cmd(cache_id: str, core_id: str = None, per_io_class: bool
         command += (" -f " if shortcut else " --filter ") + filter
     if output_format is not None:
         command += (" -o " if shortcut else " --output-format ") + output_format
+    if full_path:
+        command += (" -p " if shortcut else " --full-path ")
```

Comment:
shouldn't it be '-b'? I looked into casadm help:  
-b  --by-id-path                    Display by-id path to disks instead of short form /dev/sdx